og_format_examples_files/sg558_20240206T000000_R.cdl

```diff
@@ -0,0 +1,549 @@
+		INSTRUMENT_DISSOLVED\ GAS\ SENSORS_590:type = "dissolved gas sensors" ;
+		INSTRUMENT_DISSOLVED\ GAS\ SENSORS_590:type_vocabulary = "http://vocab.nerc.ac.uk/collection/L05/current/351/" ;
+		INSTRUMENT_DISSOLVED\ GAS\ SENSORS_590:maker = "Aanderaa" ;
+		INSTRUMENT_DISSOLVED\ GAS\ SENSORS_590:maker_vocabulary = "http://vocab.nerc.ac.uk/collection/L35/current/MAN0007/" ;
+		INSTRUMENT_DISSOLVED\ GAS\ SENSORS_590:model = "Oxygen optode 4831" ;
+		INSTRUMENT_DISSOLVED\ GAS\ SENSORS_590:model_vocabulary = "https://vocab.nerc.ac.uk/collection/L22/current/TOOL1239" ;
+		INSTRUMENT_DISSOLVED\ GAS\ SENSORS_590:serial_number = "590" ;
+		INSTRUMENT_DISSOLVED\ GAS\ SENSORS_590:calibration_date = "" ;
+	int64 INSTRUMENT_WATER\ TEMPERATURE\ SENSOR_0363 ;
+		INSTRUMENT_WATER\ TEMPERATURE\ SENSOR_0363:_FillValue = -1L ;
+		INSTRUMENT_WATER\ TEMPERATURE\ SENSOR_0363:long_name = "Seaglider CT sail 0363" ;
+		INSTRUMENT_WATER\ TEMPERATURE\ SENSOR_0363:type = "water temperature sensor" ;
+		INSTRUMENT_WATER\ TEMPERATURE\ SENSOR_0363:type_vocabulary = "http://vocab.nerc.ac.uk/collection/L05/current/" ;
+		INSTRUMENT_WATER\ TEMPERATURE\ SENSOR_0363:maker = "Sea-Bird Scientific" ;
+		INSTRUMENT_WATER\ TEMPERATURE\ SENSOR_0363:maker_vocabulary = "http://vocab.nerc.ac.uk/collection/L35/current/MAN0013/" ;
+		INSTRUMENT_WATER\ TEMPERATURE\ SENSOR_0363:model = "Unpumped CT sail CTD" ;
+		INSTRUMENT_WATER\ TEMPERATURE\ SENSOR_0363:model_vocabulary = "https://vocab.nerc.ac.uk/collection/L22/current/TOOL1188" ;
+		INSTRUMENT_WATER\ TEMPERATURE\ SENSOR_0363:serial_number = "0363" ;
+		INSTRUMENT_WATER\ TEMPERATURE\ SENSOR_0363:calibration_date = "" ;
+	int64 INSTRUMENT_DATA\ LOGGERS_sg558 ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:_FillValue = -1L ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:long_name = "Seaglider logger sg558" ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:type = "data loggers" ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:type_vocabulary = "http://vocab.nerc.ac.uk/collection/L05/current/DLOG/" ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:maker = "Kongsberg Maritime" ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:maker_vocabulary = "http://vocab.nerc.ac.uk/collection/L35/current/MAN0015/" ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:model = "Seaglider M1 Glider data logger" ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:model_vocabulary = "https://vocab.nerc.ac.uk/collection/L22/current/TOOL1185" ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:serial_number = "sg558" ;
+		INSTRUMENT_DATA\ LOGGERS_sg558:calibration_date = "" ;
+	string TRAJECTORY ;
+		TRAJECTORY:cf_role = "trajectory_id" ;
+		TRAJECTORY:long_name = "trajectory name" ;
+		TRAJECTORY:data_mode_vocabulary = "" ;
+	string PLATFORM_TYPE ;
+		PLATFORM_TYPE:long_name = "type of glider" ;
+		PLATFORM_TYPE:platform_type_vocabulary = "" ;
+	string PLATFORM_MODEL ;
+		PLATFORM_MODEL:long_name = "model of the glider" ;
+		PLATFORM_MODEL:platform_model_vocabulary = "" ;
+	string WMO_IDENTIFIER ;
+		WMO_IDENTIFIER:long_name = "wmo id" ;
+	string DEPLOYMENT_TIME ;
```

Comment:
Format describes DEPLOYMENT_TIME as a double, not string

Reply:
ah yes good spot. I think we had this as a string to keep human readable more like a metadata field than a data field but it is easier for a machine to work with.